vm/devices/storage/disk_nvme/nvme_driver/fuzz/fuzz_main.rs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#![cfg_attr(all(target_os = "linux", target_env = "gnu"), no_main)]
+
+//! A 2-way fuzzer developed to fuzz the nvme driver from the Guest side with arbitrary driver
+//! action and from the Host side with arbitrary responses from the backend.
+mod fuzz_emulated_device;
+mod fuzz_nvme_driver;
+
+use crate::fuzz_nvme_driver::FuzzNvmeDriver;
+
+use arbitrary::Unstructured;
+use lazy_static::lazy_static;
+use pal_async::DefaultPool;
+use std::sync::Mutex;
+use xtask_fuzz::fuzz_target;
+
+// Input bytes we want to use
+const MIN_INPUT:usize=200;
+
+// Use lazy_static to allow swapping out underlying vector
```

Comment:
Suggest you promote these comments to a higher level. By that I mean: this (and the following `get_raw_data`) are here to work around the compiler's inability to track the async closure's lifetimes. The most interesting thing to document is why you need this and how you can trust that we're not leaking memory.

Reply:
I am not sure what you mean by "higher level". Do you mean making this a doc comment instead?